plugins/source/pagerduty/codegen/recipes/templates/all_tables.go.tpl

```diff
@@ -0,0 +1,18 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package plugin
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+{{- range $resource := . }}
```

Comment:
```suggestion
{{- range . }}
"github.com/cloudquery/cloudquery/plugins/source/pagerduty/resources/services/{{ .SubService }}"
{{- end }}
```

Reply:
This is copy-pasted from the AWS codegen - and I like the clarity of giving a custom name to the loop variable